mriqc/reports/utils.py

```diff
@@ -109,6 +109,78 @@ def read_report_snippet(in_file):
             corrected.append(line)
         return '\n'.join(corrected[svg_tag_line:])
 
+def upload_qc_metrics(in_iqms, email = '', no_sub = False):
+    """Upload qc metrics to remote repository.
+
+    Arguments:
+    in_iqms -- Path to the qc metric json file as a string
+
+    Keyword arguments:
+    email -- email address to be included with the metric submission, defaults to empty string
+    no_sub -- Flag from settings indicating whether or not metrics should be submitted. If False, metrics will be submitted. If True, metrics will not be submitted. Defaults to False.
+
+    Returns:
+    either returns response object if a response was successfully sent
+    or it returns the string "No Response"
+    """
+    from json import load,dumps
+    import requests
+    from mriqc import logging
+
+    report_log = logging.getLogger('mriqc.report')
+    report_log.setLevel(logging.INFO)
+
+    if no_sub == True:
+        report_log.info('QC metrics were not uploaded because --no_sub or --testing options were set.')
+        r = "No Response"
+    else:
+        with open(in_iqms, 'r') as h:
+            in_data = load(h)
+        #metadata whitelist
+        whitelist = [u"ContrastBolusIngredient",u"RepetitionTime",u"TaskName",u"Manufacturer",
```

Comment:
MRIQC is now Python 3 only. Please remove the incompatible `u-` prefixes to these strings.